charts/rancher-turtles/templates/post-upgrade-job.yaml

```diff
@@ -23,13 +23,6 @@ rules:
   verbs:
   - list
   - delete
-- apiGroups:
```

Comment:
I was looking into git history to understand this better and it seems the particular piece of code was introduced in https://github.com/rancher/turtles/pull/581/commits/75d682ea81ce012ba8c00e2512c847d741ea65eb to give a possibility to opt-in for cluster downgrade scenario with managementv3-cluster-migration. 

And since we enabled v3 migration by default recently, look like it was **partially** removed in https://github.com/rancher/turtles/pull/979/commits/1f6972edaea0c05de2406a9bd3020f8171a67576#diff-c0588fa3a386b0b96a535569bdd77004aa57407c73e52c49502cfd52a46eb23dL1 and remaining parts are now cleaned-up with this patch. Does it sound an accurate summary @salasberryfin ?